### ISSUE TITLE
ImportError: cannot import name 'tags_to_entities'

### ISSUE THREAD

**Issue by Melgeorges** [explosion/spaCy#1132]
Hello,
I just discovered the package, and it seems really interesting but cannot get through the installation,

When importing in Notebooks, I get the following error:

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-75-c080f6458562> in <module>()
----> 1 import spacy

/home/melanie/anaconda3/lib/python3.6/site-packages/spacy/__init__.py in <module>()
      3 
      4 from . import util
----> 5 from .deprecated import resolve_model_name
      6 from .cli import info
      7 

/home/melanie/anaconda3/lib/python3.6/site-packages/spacy/deprecated.py in <module>()
      6 from . import about
      7 from . import util
----> 8 from .cli import download
      9 from .cli import link
     10 

/home/melanie/anaconda3/lib/python3.6/site-packages/spacy/cli/__init__.py in <module>()
      3 from .link import link
      4 from .package import package
----> 5 from .train import train, train_config
      6 from .model import model
      7 from .convert import convert

/home/melanie/anaconda3/lib/python3.6/site-packages/spacy/cli/train.py in <module>()
      6 
      7 from ..util import ensure_path
----> 8 from ..scorer import Scorer
      9 from ..gold import GoldParse, merge_sents
     10 from ..gold import read_json_file as read_gold_json

/home/melanie/anaconda3/lib/python3.6/site-packages/spacy/scorer.py in <module>()
      2 from __future__ import division, print_function, unicode_literals
      3 
----> 4 from .gold import tags_to_entities
      5 
      6 

ImportError: cannot import name 'tags_to_entities'

## Your Environment

python -m spacy info --markdown : 
ImportError: /home/melanie/anaconda3/lib/python3.6/site-packages/spacy/vocab.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZTINSt8ios_base7failureB5cxx11E

* Operating System: Mint
* Python Version Used: Python 3.6.0 :: Anaconda 4.3.1 (64-bit)

**Comment by pvcastro** [comment]
Same problem, same environment

**Comment by ravinarayanaA** [comment]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\ProgramData\Anaconda2\lib\site-packages\spacy\__init__.py", line 5, i
n <module>
    from .deprecated import resolve_model_name
  File "C:\ProgramData\Anaconda2\lib\site-packages\spacy\deprecated.py", line 8,
 in <module>
    from .cli import download
  File "C:\ProgramData\Anaconda2\lib\site-packages\spacy\cli\__init__.py", line
5, in <module>
    from .train import train, train_config
  File "C:\ProgramData\Anaconda2\lib\site-packages\spacy\cli\train.py", line 8,
in <module>
    from ..scorer import Scorer
  File "C:\ProgramData\Anaconda2\lib\site-packages\spacy\scorer.py", line 4, in
<module>
    from .gold import tags_to_entities
ImportError: DLL load failed: The application has failed to start because its si
de-by-side configuration is incorrect. Please see the application event log or u
se the command-line sxstrace.exe tool for more detail.

Environment: windows OS, Python 2.7.13 |Anaconda 4.4.0 (64-bit).

**Comment by groxli** [comment]
I see the same behavior with OS X. Version details:

3.5.2 |Anaconda 4.2.0 (x86_64)| (default, Jul  2 2016, 17:52:12) 
[GCC 4.2.1 Compatible Apple LLVM 4.2 (clang-425.0.28)]

**Comment by groxli** [comment]
I found one possible solution. As of 1.9.0 there is alpha support for Japanese with spaCy.  I uninstalled spaCy with pip and did a pip reinstall of spacy. (The reason I did this was because I had messed up some dependencies trying to get spacy to work with Japanese.)

In iPython, was was then able to run the following:

```
In [1]: import spacy
   ...: parser = spacy.load('ja', tagger=None, parser=None, entity=None)
   ...: text1 = '明日は暑いでしょう。'
   ...: 

    Warning: no model found for 'ja'

    Only loading the 'ja' tokenizer.


In [2]: parsed=parser.tokenizer(text1)

In [3]: for i, token in enumerate(parsed):
   ...:     print(i, token)
   ...:     
0 明日
1 は
2 暑い
3 でしょ
4 う
5 。
```
One side note: It doesn't appear there currently any Japanese models included (or I simply haven't figured out how to download/install the Japanese models.) When I try and run the following command in a terminal:

`python -m spacy download ja`

I get the following message:

```
Compatibility error

No compatible model found for 'ja' (spaCy v1.9.0).
```
Even without a spacy model, you can still take advantage of the parsing and tokenizing.

**Comment by crystosis** [comment]
Same problem 

Operating System: Winows 10 64
Python Version Used: Python 3.6.2 :: Anaconda (64-bit)

**Comment by santoshaditham** [comment]
Same problem on my Mac

$ python3 -m spacy info --markdown

## Info about spaCy

* **spaCy version:** 1.9.0
* **Platform:** Darwin-16.7.0-x86_64-i386-64bit
* **Python version:** 3.6.0
* **Installed models:** en

**Comment by ines** [comment]
Sorry for only getting to this now!

This problem should be fixed in the new version, v2.0. We're currently waiting to get it built for conda (it's just taking a bit longer because of Travis errors and the OSX backlogs 😓). 

Once the version is up, could you try again? [See here](https://spacy.io/usage/v2) for details on what's new in v2.0. If there are still problems with conda install and the new version, feel free to reopen.
